src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,27 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			$args['meta_query'] = array(
+				'relation' => 'OR',
+				array(
+					'key' => '_tribe_aggregator_source_name',
+					'value' => $_REQUEST['s'],
+					'compare' => 'LIKE'
+				),
+				array(
+					'key' => '_tribe_aggregator_import_name',
+					'value' => $_REQUEST['s'],
+					'compare' => 'LIKE'
+				),
+				array(
+					'key' => '_tribe_aggregator_source',
+					'value' => $_REQUEST['s'],
+					'compare' => 'LIKE'
+				)
```

Comment:
:no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: aa3bfbabb947dd99dbee7ce5634da777</sub>